changelog/minimal_runtime.dd

```diff
@@ -0,0 +1,107 @@
+Using D with no/minimal/custom runtime implementation in a pay-as-you-go fashion
+
+$(P DMD has been further decoupled from the runtime so it is now easier and more
+convenient to use D without the runtime in a pay-as-you-go fashion.  This will
+be of interest to users wishing to incrementally or partially port D to new
+platforms, users targeting bare-metal or resource constrained platforms, and
+users wishing to use D as a library from other languages without the runtime.)
```

Comment:
FYI: the changelog script magic is smart enough to auto-insert `$(P)` (i.e. make it feel like it's Markdown-flavored Ddoc) here - though it doesn't hurt to have.